### PR TITLE
Add litecoin-core@0.21.2.1

### PR DESCRIPTION
**Pull request by alexandremandim**
We're adding the new version [0.21.2.1 ](https://github.com/litecoin-project/litecoin/releases/tag/v0.21.2).

Added new key identifier [0x3620e9d387e55666 ](https://pgp.mit.edu/pks/lookup?op=get&search=0x3620E9D387E55666)(davidburkett38’s key) to Dockerfile to verify the integrity of the released binaries.

### CODE REVIEW

File: 0.21/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+RUN useradd -r litecoin \
+  && apt-get update -y \
+  && apt-get install -y curl gnupg \
+  && apt-get clean \
+  && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* \
+  && set -ex \
+  && for key in \
+    3620E9D387E55666 \
+    B42F6819007F00F88E364FD4036A9C25BF357DD4 \
+    FE3348877809386C \
```
Comment (pedrobranco) [inline 0.21/Dockerfile]: Remove unnecessary key.

File: 0.21/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+RUN useradd -r litecoin \
+  && apt-get update -y \
+  && apt-get install -y curl gnupg \
+  && apt-get clean \
+  && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* \
+  && set -ex \
+  && for key in \
+    3620E9D387E55666 \
+    B42F6819007F00F88E364FD4036A9C25BF357DD4 \
```
Comment (pedrobranco) [inline 0.21/Dockerfile]:
```suggestion
    3620E9D387E55666 # litecoin key \
    B42F6819007F00F88E364FD4036A9C25BF357DD4 # gosu key \
```

File: 0.21/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+ENV LITECOIN_DATA=/home/litecoin/.litecoin
+
+RUN curl -SLO https://download.litecoin.org/litecoin-${LITECOIN_VERSION}/linux/litecoin-${LITECOIN_VERSION}-x86_64-linux-gnu.tar.gz \
+  && curl -SLO https://download.litecoin.org/litecoin-${LITECOIN_VERSION}/SHA256SUMS.asc \
+  && gpg --verify SHA256SUMS.asc \
+  && grep $(sha256sum litecoin-${LITECOIN_VERSION}-x86_64-linux-gnu.tar.gz | awk '{ print $1 }') SHA256SUMS.asc \
+  && tar --strip=2 -xzf *.tar.gz -C /usr/local/bin \
+  && rm *.tar.gz
+
+
```
Comment (pedrobranco) [inline 0.21/Dockerfile]: Remove extra newline.

**Review by pedrobranco — CHANGES_REQUESTED** [review]

**Comment by alexandremandim** [comment]
Do u want me to squash the 2 commits into 1?

**Review by pedrobranco — APPROVED** [review]

**Pull request merged**